<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentCreation</class>
 <widget class="QWidget" name="ContentCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="KDE::DoNotExtract">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="currentCategoryLabel">
     <property name="text">
      <string>Select the category of the new content:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="category"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name of the content:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="name"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="description"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Changelog:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="changelog"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Version:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="version"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Download link 1:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="link1"/>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="link2"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Download link 2:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QComboBox" name="distroType_1"/>
        </item>
        <item row="6" column="2">
         <widget class="QComboBox" name="distroType_2"/>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="license"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>License:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Homepage:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="homepage"/>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="blog"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Blog:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QCheckBox" name="donations">
          <property name="text">
           <string>Accept Donations</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="contentId">
          <property name="text">
           <string>New Content</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Current Item:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
      <item>
       <widget class="QLabel" name="currentCategoryLabel_2">
        <property name="text">
         <string> I ensure that this content does not violate any existing copyright, law or trademark. I agree that my IP address is logged. Distributing content where you don't have the permission from the copyright holder is illegal!</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="submitButton">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="addFileGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Add File and Previews</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QPushButton" name="upload">
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="KUrlRequester" name="uploadFile">
        <property name="clickMessage">
         <string>File</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="uploadPreview1Button">
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="KUrlRequester" name="uploadPreview1">
        <property name="clickMessage">
         <string>Preview 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="KUrlRequester" name="uploadPreview2">
        <property name="clickMessage">
         <string>Preview 2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="KUrlRequester" name="uploadPreview3">
        <property name="clickMessage">
         <string>Preview 3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="uploadPreview2Button">
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="uploadPreview3Button">
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="deleteFile">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="deletePreview1">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
